web/themes/custom/my_awesome_theme/templates/block/block--views-block--comments-recent-block-1.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="recent-comments">
+  <span>
+      Recent Comments
```

Comment:
Strings should be translated with |t twig filter or by putting them in the {% trans %}
( |t is better in this case)